lambdas/get-addresses/src/services/config-service.ts

```diff
@@ -0,0 +1,37 @@
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+
+export class ConfigService {
+    constructor(private ssmClient: SSMClient) {}
+
+    private parameterPrefix = process.env.AWS_STACK_NAME || "";
+    readonly config: { addressLookupStorageTableName: string | undefined } = {
+        addressLookupStorageTableName: undefined,
+    };
+
+    public async init(): Promise<void[]> {
+        // how do handle the parameter names
+        const getParameterAddressQueryTable = this.getParameter(
+            `${this.parameterPrefix}/GetAddressQueryTable`,
+            "addressLookupStorageTableName",
```

Comment:
This can be simplified if we use the same name for the parameter store and what gets stored as a key in the `config` variable. Opinions?